sql/core/src/test/resources/sql-tests/inputs/group-by.sql

```diff
@@ -53,3 +53,13 @@ set spark.sql.groupByAliases=false;
 
 -- Check analysis exceptions
 SELECT a AS k, COUNT(b) FROM testData GROUP BY k;
+
+-- turn on group by aliases (for remaining tests)
+set spark.sql.groupByAliases=true;
```

Comment:
This is undoing the opposite change on line 52; I added this here instead of adding my tests above because I wanted to minimize the diff / renumbering of existing test cases.